src/components/Form.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import { useState } from 'react';
+
+type FormProps = {
+  onSubmit: (e: React.FormEvent<HTMLFormElement>) => void;
+};
+
+export default function Form({ onSubmit }: FormProps) {
+  const [inputValue, setInputValue] = useState('');
+
+  return (
+    <form onSubmit={onSubmit} className="flex flex-col gap-4 w-80">
+      <input
+        type="text"
```

Comment:
[IMO]
[作ってくださった関数](https://github.com/Kazuki0320/sns-scouter/blob/main/src/app/calc/ScouterCalculator.tsx)的に、type は number かなと思いました！

number にすると、min="0" で簡単なバリデーションもできます
```suggestion
        type="number"
        min="0"
```
<img width="427" alt="スクリーンショット 2025-03-15 16 06 35" src="https://github.com/user-attachments/assets/8878f7bc-f3e4-4813-9615-208e02c8d9df" />

Reply:
[Q]
こちら、今の所 text のままにされているかと思いますが、やっぱり text として処理する感じになりましたか？

またその場合、src/app/globals.css の`@layer utilities {`の記載と、input タグにある`className="p-2 border-gray-300 rounded-md text-black appearance-none no-spinner"`のappearance-none no-spinnerは全て不要かなと思います！